mod-source-record-storage-server/src/main/java/org/folio/dao/RecordDaoImpl.java

```diff
@@ -491,6 +494,68 @@ private void appendJoinAlternative(SelectJoinStep<?> selectJoinStep, ParseLeader
     }
   }
 
+  @Override
+  public Future<RecordsIdentifiersCollection> getMatchedRecordsIdentifiers(MatchField matchedField, TypeConnection typeConnection,
+                                                                           boolean externalIdRequired, int offset, int limit, String tenantId) {
+    Table<org.jooq.Record> marcIndexersPartitionTable = table(name(MARC_INDEXERS_PARTITION_PREFIX + matchedField.getTag()));
```

Comment:
Is it possible that more than one tag would need to be matched?

Reply:
Currently, such a scenario is not expected since a match profile supports only one matching criteria.